### DISCUSSION TITLE
Adjust the time/distance to a speed camera

### DISCUSSION THREAD

**Discussion by alensiljak** [osmandapp/OsmAnd#17837] [Q&A] [answered]
Hi! Is it possible to adjust the time (or distance) to the next speed camera? I find that, with offline maps, the announcement comes quite late, usually when the camera is next to the car or after. I've noticed this while driving through a (foreign) country/region where the speed limit information was non-existent and/or wrong.
An earlier warning, with the speed limit (if existing), would be quite helpful. I've noticed that there are quite a number of areas where the speed limit signs/areas are confusing.

**Comment by DmitryAlexei** [comment, marked as answer]
Please see https://osmand.net/docs/user/navigation/guidance/voice-navigation#announcement-time

**Comment by alensiljak** [comment]
Thank you! I must have been blind as I was looking at these options several times.

The full path to the setting is, i.e.:
Menu -> Configure Profile -> Navigation settings -> Voice Prompts (tap on the item) -> Announcement Time
